flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcTableSourceSinkFactoryTest.java

```diff
@@ -255,6 +255,17 @@ public void testJdbcValidation() {
 			fail("exception expected");
 		} catch (IllegalArgumentException ignored) {
 		}
+
+		// lookup max-retries properties not complete
```

Comment:
// lookup max-retries property less than zero

Reply:
Please fix the comment